docs/01-start-here/06-testing-tips.md

```diff
@@ -42,6 +49,8 @@ To run only the tests for a particular test suite, use the command:
   test-only *<name-of-your-test-suite>
 ```
 
+## Testing AMP
```

Comment:
Is this worth having in this doc, or should it be combined with: https://github.com/guardian/frontend/blob/master/docs/03-dev-howtos/17-working-with-amp.md